optima/optimization.py

```diff
@@ -586,14 +586,14 @@ def minmoney(name=None, project=None, optim=None, inds=None, tvec=None, verbose=
         ##########################################################################################################################
         args['which'] = 'outcomes'
         budgetvec3, fval, exitflag, output = asd(objectivecalc, budgetvec, args=args, xmin=xmin, timelimit=maxtime, MaxIter=maxiters, verbose=verbose)
-        budgetvec4 = constrainbudget(origbudget=origbudget, budgetvec=budgetvec1, totalbudget=args['totalbudget'], budgetlims=optim.constraints, optiminds=optiminds, outputtype='vec')
+        budgetvec4 = constrainbudget(origbudget=origbudget, budgetvec=budgetvec3, totalbudget=args['totalbudget'], budgetlims=optim.constraints, optiminds=optiminds, outputtype='vec')
```

Comment:
i get a bit confused with the budgetvec1, budgetvec2, budgetvec3, etc... would it be possible to give them more descriptive names at some point?